src/sql/workbench/contrib/notebook/browser/htmlMarkdownConverter.ts

```diff
@@ -295,3 +307,16 @@ export function addHighlightIfYellowBgExists(node, content: string): string {
 	}
 	return content;
 }
+
+export function getUriAnchorLink(node, notebookUri: URI): URI {
+	let notebookLink: URI;
+	const sectionLinkToAnotherFile = node.href.includes('#') && !node.attributes.href?.nodeValue.startsWith('#');
```

Comment:
Stuff like this is why typings are nice - what's the difference between node.href and node.attributes.href.nodeValue?